ztools/x509/extensions.go

```diff
@@ -128,6 +128,298 @@ type ExtendedKeyUsage []ExtKeyUsage
 
 type CertificatePolicies []asn1.ObjectIdentifier
 
+// The string functions for CertValidationLevel are auto-generated via
+// `go generate <full_path_to_x509_package>` or running `go generate` in the package directory
+//go:generate stringer -type=CertValidationLevel -output=generated_certvalidationlevel_string.go
+type CertValidationLevel int
```

Comment:
I like level because it suggests a linear, tiered classification, rather than a generic classification label. @zakird @dadrian thoughts?